roman_numeral_converter_spec.rb

```diff
@@ -0,0 +1,70 @@
+require_relative "./roman_numeral_converter.rb"
+
+describe "Roman Numeral Converter" do
```

Comment:
:+1: This way we have access to `described_class` here too; Which is handy incase we want to change the class under test